src/clj/harja/kyselyt/tehtavamaarat.sql

```diff
@@ -120,7 +120,8 @@ FROM tehtavaryhma tr1
        JOIN tehtavaryhma tr3 ON tr2.id = tr3.emo
                                     and tr3.nimi not like ('%Lisätyöt%')
                                     -- Jätetään tehtäväryhmä: Tilaajan rahavaraus (T3) pois, koska se lisätään bonuksista
-                                    and tr3.yksiloiva_tunniste != '0e78b556-74ee-437f-ac67-7a03381c64f6'
+                                    AND (tr3.yksiloiva_tunniste IS NULL
+                                        OR (tr3.yksiloiva_tunniste IS NOT NULL AND tr3.yksiloiva_tunniste != '0e78b556-74ee-437f-ac67-7a03381c64f6'))
```

Comment:
Tässähän tuo ensimmäinen IS NOT NULL ehto on turha, kun nulluus on jo tsekattu

(tr3.yksiloiva_tunniste IS NOT NULL AND

Reply:
Kyllä. Mutta laitoin sen silti :) koska se tuntui järkevämmältä ja helpotti mun ymmärtämistä, että miksi tämä aiheutti ongelmia.